### PR TITLE
Add tencent-plugins english version docs

### PR DESCRIPTION
**Pull request by tinafangkunding**
## What did you implement

Add tencent-scf plugin's english docs to serverless framework

## How can we verify it

the tree of docs list like this:
```
|_ /docs/providers/tencent
  |_ /cli-reference
  |_ /events
  |_ /guide
  |_ README.md
```

## Todos

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO

### CODE REVIEW

File: docs/providers/tencent/guide/deploying.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Deploy All
+
+This is the main method for doing deployments with the Serverless Framework:
+
+```bash
+serverless deploy
+```
+
+Use this method when you have updated your Function, Event or Resource configuration in `serverless.yml` and you want to deploy that change (or multiple changes at the same time) to Amazon Web Services.
```
Comment (pmuens) [inline docs/providers/tencent/guide/deploying.md]: Is it correct that we're referring "Amazon Web Services" here? Or should it be "Tencent Cloud"?

Comment (tinafangkunding) [reply docs/providers/tencent/guide/deploying.md]: > Is it correct that we're referring "Amazon Web Services" here? Or should it be "Tencent Cloud"?

I'm so sorry about that, it should be Tencent Cloud here.

**Comment by tinafangkunding** [comment]
Thanks a lot for run prettify to format the docs @pmuens

File: docs/providers/README.md [hunk 2]
```diff
@@ -78,6 +78,13 @@ Under the hood, the Serverless Framework is deploying your code to a cloud provi
     </div>
   </div>
   </a>
+  <a href="./tencent/">
+  <div class="providerSection">
+    <div class="providerSectionHeader">
+        <img src="https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/docs/tencent-cloud-logo-gray.png" width="250" draggable="false" class='tencent-logo'/>
```
Comment (pmuens) [inline docs/providers/README.md]: @pdaryani here's the part where you can find the URL to the logo and the class which formats it.

File: docs/providers/tencent/cli-reference/install.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+- `--url` or `-u` The services Git URL. **Required**.
+- `--name` or `-n` Name for the service.
+
+## Examples
+
+### Installing a service from a GitHub URL
+
+```bash
+serverless install --url https://github.com/tencentcloud/serverless/tencent-nodejs
```
Comment (pdaryani) [inline docs/providers/tencent/cli-reference/install.md]: Hi @tinafangkunding , this link seems to be broken, could you please help fix this?

Comment (tinafangkunding) [reply docs/providers/tencent/cli-reference/install.md]: > Hi @tinafangkunding , this link seems to be broken, could you please help fix this?

Yes sure, I'll fix that ASAP

Comment (tinafangkunding) [reply docs/providers/tencent/cli-reference/install.md]: Already fix that, thanks!

**Comment by tinafangkunding** [comment]
Thank you all guys!

**Pull request merged**